### PR TITLE
Fix sort list and fix add edit

### PR DESCRIPTION
**Pull request by mvn-vyle-dn**
## Description
- Sort show list in Category, Store, Order
- Add old() when create, update
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-hienpham** [comment]
LGTM

File: app/Http/Controllers/Admin/OrderController.php [hunk 1]
```diff
@@ -16,10 +17,45 @@ class OrderController extends Controller
...
+     * Display the specified resource.
+     *
+     * @param int $id int
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $order = Order::with('user')->find($id);
+        $detailOrders = Order::find($id)->orderDetails()->with('product:id,name,price')->get();
```
Comment (at-quangtran) [inline app/Http/Controllers/Admin/OrderController.php]: chỗ này sửa lại rồi mà? dùng with với cái ở trên

Comment (at-vyle) [reply app/Http/Controllers/Admin/OrderController.php]: là do chưa pull master về á anh

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**